mediawiki.py

```diff
@@ -11,9 +11,9 @@
 from pprint import pprint
 
 class Importer(object):
-    def __init__(self, api_url, http_user=None, http_pass="", wiki_user=None, wiki_pass="", verbose=False):
+    def __init__(self, api_url, http_user=None, http_pass="", wiki_user=None, wiki_pass="", wiki_domain="", verbose=False):
         self.verbose = verbose
-        self.mw = simplemediawiki.MediaWiki(api_url,http_user=http_user,http_password=http_pass)
+        self.mw = simplemediawiki.MediaWiki(api_url, http_user=http_user, http_password=http_pass, domain=wiki_domain)
```

Comment:
I still can't merge this as-is because this line will crash if the user has an unpatched mediawiki install (the domain= argument won't exist.)

Can you change the code so it only passes `domain=` if that argument is set?